### DISCUSSION TITLE
Horizontal scroll at zoom

### DISCUSSION THREAD

**Discussion by undefineq** [ValentinH/react-easy-crop#224] [Q&A] [answered]
I'm not sure if it's a bug. In all the presented sandboxes, when the image is zoomed in, a horizontal scroll appears in all browsers except Safari.

Environment
-
Mac OS, Firefox 

Steps To Reproduce
-
1. Open [Basic example with hooks](https://codesandbox.io/s/v69ly910ql)
2. Open browser preview in new window 
3. Try to zoom

<img width="500" alt="Снимок экрана 2021-03-25 в 19 40 41" src="https://user-images.githubusercontent.com/13893221/112510156-3b0f2a00-8da2-11eb-9c87-5aabb720c623.png">

**Comment by undefineq** [comment, marked as answer]
Sorry fo noise)
I was able to fix it myself. This behavior was caused by the slider.

**Reply by ValentinH** [reply]
Could you please share what you did for the fix? If it's happening on the sandboxes used for the demos, it makes sense to apply the fix on them. 😊

**Reply by undefineq** [reply]
Sure, but I wouldn't want to do a disservice.

Perhaps the problem is in the slider implementation

<img width="600" alt="Снимок экрана 2021-03-26 в 17 05 36" src="https://user-images.githubusercontent.com/13893221/112643724-b7167a00-8e55-11eb-88d7-4022a9a3fdd4.png">

We can assume that `overflow: hidden` can be the right choice, but it's not. Because of this

<img width="137" alt="Снимок экрана 2021-03-26 в 17 19 36" src="https://user-images.githubusercontent.com/13893221/112645400-7c154600-8e57-11eb-85d9-05999fae11cf.png">

So I wrapped slider with a `div`, add paddings and make it `overflow: hidden`.  If the solution is suitable, I can add it.

Maybe, correct solution is to update the material-ui library to version 4. Since the `material-ui/lab` package is outdated, and was not yet ready to move to the core.

So, how can I help you?

**Reply by ValentinH** [reply]
You are right, simply updating `@material-ui` and using the Slider from core, fix the issue 🙂
